nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/cluster/nf-cluster-table.js

```diff
@@ -813,7 +813,7 @@
     function sort(sortDetails, dataView, tab) {
         // defines a function for sorting
         var comparer = function (a, b) {
-            if (sortDetails.columnId === 'heartbeat' || sortDetails.columnId === 'uptime') {
+            if (sortDetails.columnId === 'heartbeat' || sortDetails.columnId === 'nodeStartTime') {
```

Comment:
This change to `nodeStartTime` appears to be incorrect since `nodeStartTime` is a `field` value not an `id` value. Should that part of the conditional be removed since the `uptime` ID is being compared later on using the duration comparison?